### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:  Jaws and NVDA and NVDA announces additional inappropriate information when "Run on web app" dialog  is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1480]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Navigate to and select Project ->Azure->Run on web app
4. "Run on web app" dialog will open. 
5. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Run on web app" dialog is opened 

**Jaws  behaviour:**
 Run on Web App dialog
Invalid Data: Choose a web app to deploy. Name JDK Web container Resource group Artifact Before launch: Launch Web Browser, Build Artifacts, Activate tool window 
Use Existing Web App radio button checked
1 of 2
To change the selection press Up or Down Arrow.

**NVDA Behaviour:**
Run on Web App  dialog  
Invalid Data: Choose a web app to deploy.
Artifact
Please check the web.config file used to deploy this non-WAR executable.
Before launch: Activate tool window
Use Existing Web App  radio button  checked  1 of 2
panel
combo box  collapsed
list
list item  selected  1 of 26

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e  Run on Web App dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app-opened](https://user-images.githubusercontent.com/22956130/36105322-56145b56-103a-11e8-9e62-bd73d45e6b58.jpg)
![2b6f07cb](https://user-images.githubusercontent.com/22956130/36105328-593ad88c-103a-11e8-9cc6-49d5034c303d.jpg)

**Comment by Eskibear** [comment]
duplicated with #1485

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749911
